src/main/java/hudson/tasks/junit/History.java

```diff
@@ -46,10 +46,10 @@
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.stream.Collectors;
 import jenkins.util.SystemProperties;
+import org.apache.commons.math3.stat.regression.SimpleRegression;
 import org.kohsuke.accmod.Restricted;
 import org.kohsuke.accmod.restrictions.NoExternalUse;
 import org.kohsuke.stapler.bind.JavaScriptMethod;
-import umontreal.ssj.functionfit.LeastSquares;
```

Comment:
Uses libraries from `colt`, which we can avoid by using `SimpleRegression` instead.